STATUS: INLINE SUCCESSFUL
MODIFIED: de.strullerbaumann.visualee.examiner.cdi.ExaminerResourceTest.testFindAndSetAttributesResourceInstance
MODIFIED FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/ExaminerResourceTest.java
CLEANER: de.strullerbaumann.visualee.examiner.cdi.TestGroup100Case0.test258
CLEANER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/examiner/cdi/TestGroup100Case0.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.SourcecodeFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/intIDRepair/toolRepos/ODRepair/experiments/_downloads/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/SourcecodeFilterTest.java
ORIGINAL CLEANER SIZE: 18
NEW CLEANER SIZE: 2
ELAPSED TIME: 65.5569999217987
==========================
@@ -129,0 +129,4 @@
+ {
+     de.strullerbaumann.visualee.filter.boundary.FilterContainer filterContainer0 = de.strullerbaumann.visualee.filter.boundary.FilterContainer.getInstance();
+     filterContainer0.clear();
+ }
